data/json/items/ammo.json

```
@@ -804,7 +804,7 @@
     "category": "spare_parts",
     "price": 2000,
     "price_postapoc": 100,
-    "name": { "str": "medical tape" },
+    "name": { "str": "medical tape", "str_pl": "medical tape rolls" },
```

```suggestion
    "name": { "str_sp": "medical tape" },
```
That you find 35 of them at the same time indicates a low likelihood of the description being right.